packages/appstore/src/components/transaction-list/transaction-list.scss

```diff
@@ -7,13 +7,13 @@
     &__scroll {
         position: absolute;
         inset: 0;
-        padding-top: 7.8rem;
+        padding-top: 7.2rem;
 
         @include mobile() {
-            padding-top: 7rem;
+            padding-top: 6.4rem;
 
             &__crypto {
-                padding-top: 10.72rem;
+                padding-top: 10.52rem;
```

Comment:
Where is this number coming from? 😅

Reply:
The header with the filter and the "pending transactions" toggle on mobile in case of a crypto wallet has this height 😅 

<img width="396" alt="Screenshot 2023-09-14 at 14 18 17" src="https://github.com/binary-com/deriv-app/assets/119863957/db4821ab-0ec5-42ab-a681-af2d42f213a5">

Reply:
The `.2` part comes from this:

<img width="435" alt="Screenshot 2023-09-14 at 14 20 22" src="https://github.com/binary-com/deriv-app/assets/119863957/904896d8-3782-4a49-a497-0b6e91fc1659">

Reply:
Aaaand that in turn is coming from this:

<img width="414" alt="Screenshot 2023-09-14 at 14 21 40" src="https://github.com/binary-com/deriv-app/assets/119863957/6529bf3c-e425-469f-b5d3-20860adeece5">


Which is the `deriv/components` toggle's default height I guess 👀

Reply:
Is that fixed or does it change ever so slightly for different screen sizes?

Reply:
It's fixed

Reply:
Rounded the values so that we don't have 2 decimals